tools/isobuild/compiler-plugin.js

```diff
@@ -357,16 +368,36 @@ class ResourceSlot {
       throw Error("addStylesheet on non-source ResourceSlot?");
     }
 
-    self.outputResources.push({
-      type: "css",
-      refreshable: true,
-      data: new Buffer(files.convertToStandardLineEndings(options.data), 'utf8'),
-      servePath: self.packageSourceBatch.unibuild.pkg._getServePath(
-        options.path),
-      // XXX do we need to call convertSourceMapPaths here like we did
-      //     in legacy handlers?
-      sourceMap: options.sourceMap
-    });
+    if (self.packageSourceBatch.usesModules()) {
+      const data = new Buffer(
+        'require("modules/css").addStyles(' + JSON.stringify(
+          files.convertToStandardLineEndings(options.data)
+        ) + ');\n'
+      );
+
+      self.jsOutputResources.push({
+        type: "js",
+        refreshable: true, // TODO Really?
```

Comment:
I couldn't find anywhere in the code that reads this property. Did I miss something?